### ISSUE TITLE
Log Likelihood Ratio P-value incorrectly named as log-rank?

### ISSUE THREAD

**Issue by arichards2564** [kassambara/survminer#640]
I believe that p.value.log is actually referring to the log of the likelihood ratio test, whereas the p.value.sc is referring to the score log-rank test.
According to this: https://rdrr.io/cran/survival/src/R/summary.coxph.R the log LRT is stored in logtest while the log-rank is in sctest. Then when it goes through broom::glance, these are logged as p.value.log and p.value.sc respectively (https://github.com/tidymodels/broom/blob/main/R/survival-coxph-tidiers.R#L153-L155)

https://github.com/kassambara/survminer/blob/64de6746a7ee9cd758dc8d14ca88c8383e6851a6/R/ggforest.R#L172C21-L172C40

**Comment by kassambara** [comment]
You're right — thank you for the precise diagnosis and the source links. This was a genuine mislabel.

**Root cause.** `ggforest()`'s bottom caption printed `broom::glance()$p.value.log`, which is the p-value of the **likelihood ratio test** (`survival` stores the statistic in `logtest`), but labeled it "Log-Rank". The score/log-rank test is `p.value.sc` (`sctest`). Confirmed numerically on a `lung` Cox model — the two are close but distinct:

```
p.value.log (LRT / logtest) : 1.0828e-06   <- the value ggforest shows
p.value.sc  (score/log-rank): 1.0832e-06
```

**Fix (dev version 0.5.2.999, #749).** The caption is now labeled correctly as **"Global p-value (Likelihood ratio test)"**. Only the label is corrected — the value shown is unchanged (still the LRT p-value), so no existing number changes.

```r
# remotes::install_github("kassambara/survminer")
library(survival)
library(survminer)
m <- coxph(Surv(time, status) ~ age + sex + ph.ecog, data = lung)
ggforest(m, data = lung)
```

![ggforest with corrected likelihood-ratio-test caption](https://i.imgur.com/zVMpC6S.png)

Closing as fixed; reopen if anything looks off.
